tests/e2e/tests/pilot/pilot_test.go

```diff
@@ -75,93 +97,100 @@ func init() {
 		"Debug, skip clean up on failure")
 }
 
-func setup(config *tutil.Infra, t *testing.T) {
-	// Only run the tests if the user has defined the KUBECONFIG environment variable.
-	if config.KubeConfig == "" {
-		t.Skip("Env variable KUBECONFIG not set. Skipping integration tests")
-	}
-
-	if config.Hub == "" {
-		t.Skip("HUB not specified. Skipping integration tests")
-	}
-
-	if config.Tag == "" {
-		t.Skip("TAG not specified. Skipping integration tests")
-	}
-
-	// TODO(nmittler): Restore this tutil.Tlog("Deploying infrastructure", spew.Sdump(config))
-	if config.Err = config.Setup(); config.Err != nil {
-		t.Fatal(config.Err)
+func setup(infra *tutil.Infra, t *testing.T) {
+	tutil.Tlog("Deploying infrastructure", spew.Sdump(infra.Config))
```

Comment:
there is probably a subset of the previous object that can be printed without needing a new object no ?
(the diff is quite big just to reduce log flood - and yet the tests do look floody still)